Formula/libgit2.rb

```diff
@@ -28,6 +28,7 @@ def install
     args = std_cmake_args
     args << "-DBUILD_EXAMPLES=YES"
     args << "-DBUILD_CLAR=NO" # Don't build tests.
```

Comment:
This one can be cleared I guess: https://github.com/libgit2/libgit2/commit/4e14d4c6a6d4629601cc7e1659df85eeb450df10

Reply:
That is right! 👍